docs/resources/rediscloud_subscription.md

```diff
@@ -93,16 +94,16 @@ only with Redis Labs internal cloud account
 
 The `creation_plan` block supports:
 
-* `memory_limit_in_gb` - (Required) Maximum memory usage for the initial databases
+* `memory_limit_in_gb` - (Required) Maximum memory usage that will be used your databases.
 * `support_oss_cluster_api` - (Optional) Support Redis open-source (OSS) Cluster API. Default: ‘false’
 * `modules` - (Optional) a list of modules that will be used by the databases in this subscription. 
 Example: `modules = ["RedisJSON", RedisBloom"]`.
-* `replication` - (Required) Databases replication.
-* `quantity` - (Required) The planned number of databases.
-* `throughput_measurement_by` - (Required) Throughput measurement method, (either ‘number-of-shards’ or ‘operations-per-second’)
-* `throughput_measurement_value` - (Required) Throughput value (as applies to selected measurement method)
+* `replication` - (Required) Databases replication. Set to `true` if any of your databases will use replication.
+* `quantity` - (Required) The planned number of databases in the subscription.
+* `throughput_measurement_by` - (Required) Throughput measurement method that will be used by your databases, (either ‘number-of-shards’ or ‘operations-per-second’)
+* `throughput_measurement_value` - (Required) Throughput value that will be used by your databases (as applies to selected measurement method)
 * `average_item_size_in_bytes` - (Optional) Relevant only to ram-and-flash clusters. Estimated average size (measured in bytes)
-of the items stored in the database. Default: 0
+of the items stored in your databases. Default: 0
```

Comment:
Again is this value for all of the databases combined, for the average database in the subscription, or for the largest database in the subscription?